celery/utils/deprecated.py

```diff
@@ -54,7 +54,7 @@ def _inner(fun):
 
         @wraps(fun)
         def __inner(*args, **kwargs):
-            from .imports import qualname
+            from . imports import qualname
```

Comment:
Is this change actually correct? `imports` isn't a keyword in Python, wasn't this a relative import from `.imports` (meaning `from celery.utils.imports import qualname`)?